FreeRTOS/FreeRTOS-Plus-TCP
# Add Network Down event in EMACHandler Task

Description
-----------
This PR adds Network Down event in EMACHandler task.

Test Steps
-----------
Ran FreeRTOS+TCP Demo on STM32H7 NUCLEO Board.

Checklist:
----------



Related Issue
-----------
[Forum Issue](https://forums.freertos.org/t/freertos-networkdown-in-xphychecklinkstatus-with-stm32h7/16262/3)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
